modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GeneNameDescProjection_plants_conf.pm

```diff
@@ -36,6 +36,15 @@ use strict;
 use warnings;
 use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::GeneNameDescProjection_conf');
 
+sub pipeline_wide_parameters {
+    my ($self) = @_;
+    return {
+	%{$self->SUPER::pipeline_wide_parameters},    
+	'exclude_species' => ['ciona_savignyi', 'homo_sapiens'],	    
```

Comment:
Is it easy to call it `antispecies` for consistency with our usual SpeciesFactory parameters?

Reply:
compara team provided with this param (exclude_species) to filter the species from GenomeDBFactory.pm(https://github.com/Ensembl/ensembl-compara/pull/322)

Reply:
Hmm ok, could we do something like: our pipeline param is named "antispecies", then passed to theirs with their naming convention?

Reply:
Although human and sea squirt are the only two that cause errors, there are 3 other non-vert outgroup species, which it probably makes sense to exclude, for consistency (I don't _think_ it would be possible to project gene names to them, but might as well make doubly sure...): caenorhabditis_elegans, drosophila_melanogaster, saccharomyces_cerevisiae

Reply:
added additional outgroup species as suggested